intro-HPC/ch_running_batch_jobs.tex

```diff
@@ -40,6 +40,494 @@ \chapter{Running batch jobs}
+prepend_path("LIBRARY_PATH","/apps/gent/CO7/sandybridge/software/GSL/2.4-GCCcore-6.4.0/lib")
+prepend_path("MANPATH","/apps/gent/CO7/sandybridge/software/GSL/2.4-GCCcore-6.4.0/share/man")
+prepend_path("PATH","/apps/gent/CO7/sandybridge/software/GSL/2.4-GCCcore-6.4.0/bin")
+prepend_path("PKG_CONFIG_PATH","/apps/gent/CO7/sandybridge/software/GSL/2.4-GCCcore-6.4.0/lib/pkgconfig")
+setenv("EBROOTGSL","/apps/gent/CO7/sandybridge/software/GSL/2.4-GCCcore-6.4.0")
+setenv("EBVERSIONGSL","2.4")
+setenv("EBDEVELGSL","/apps/gent/CO7/sandybridge/software/GSL/2.4-GCCcore-6.4.0/easybuild/GSL-2.4-GCCcore-6.4.0-easybuild-devel")
+\end{prompt}
+\ifbrussel
+\begin{prompt}
+------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
+   /apps/brussel/CO7/magnycours-ib/modules/2017b/all/GSL/2.4-GCCcore-6.4.0:
+------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
+whatis("Description:   The GNU Scientific Library (GSL) is a numerical library for C and C++  programmers.  The library provides a wide range of mathematical routines  such as random number generators, special functions and least-squares fitting.  ")
+whatis("Homepage: http://www.gnu.org/software/gsl/ ")
+conflict("GSL")
+load("GCCcore/6.4.0")
+prepend_path("ACLOCAL_PATH","/apps/brussel/CO7/magnycours-ib/software/GSL/2.4-GCCcore-6.4.0/share/aclocal")
+prepend_path("CPATH","/apps/brussel/CO7/magnycours-ib/software/GSL/2.4-GCCcore-6.4.0/include")
+prepend_path("LD_LIBRARY_PATH","/apps/brussel/CO7/magnycours-ib/software/GSL/2.4-GCCcore-6.4.0/lib")
+prepend_path("LIBRARY_PATH","/apps/brussel/CO7/magnycours-ib/software/GSL/2.4-GCCcore-6.4.0/lib")
+prepend_path("MANPATH","/apps/brussel/CO7/magnycours-ib/software/GSL/2.4-GCCcore-6.4.0/share/man")
+prepend_path("PATH","/apps/brussel/CO7/magnycours-ib/software/GSL/2.4-GCCcore-6.4.0/bin")
+prepend_path("PKG_CONFIG_PATH","/apps/brussel/CO7/magnycours-ib/software/GSL/2.4-GCCcore-6.4.0/lib/pkgconfig")
+setenv("EBROOTGSL","/apps/brussel/CO7/magnycours-ib/software/GSL/2.4-GCCcore-6.4.0")
+setenv("EBVERSIONGSL","2.4")
+setenv("EBDEVELGSL","/apps/brussel/CO7/magnycours-ib/software/GSL/2.4-GCCcore-6.4.0/easybuild/GSL-2.4-GCCcore-6.4.0-easybuild-devel")
+help([[
+
+Description
+===========
+The GNU Scientific Library (GSL) is a numerical library for C and C++
+ programmers.  The library provides a wide range of mathematical routines
+ such as random number generators, special functions and least-squares fitting.
+
+
+More information
+================
+ - Homepage: http://www.gnu.org/software/gsl/
+
+]])
+\end{prompt}
+\fi
+
+
+If you're not sure what all of this means: don't worry, you don't have to know;
+just load the module and try to use the software.
```

Comment:
Just highlight how people can see which extensions/extra Python packages are installed as a part of Python...